go/common/sgx/pcs/http.go

```diff
@@ -120,6 +119,9 @@ func (hc *httpClient) GetTCBBundle(ctx context.Context, fmspc []byte) (*TCBBundl
 
 	// Then fetch QE identity.
 	u = hc.getUrl(pcsAPIGetQEIdentityPath)
+	q = u.Query()
+	q.Set("update", string(update))
```

Comment:
I'm still not a fan of introducing a new type for everything, because than you have to cast like here.